src/IncrementalChecker.ts

```diff
@@ -211,8 +214,8 @@ export class IncrementalChecker {
       this.programConfig,
       path.dirname(this.programConfigFile),
       this.files,
-      this.watcher,
-      this.program
+      this.watcher!,
```

Comment:
I did it here because `loadVueProgram` is called by `nextIteration` which ensures that `watcher` is set, it felt kinda silly to add an assertion, I can change that if you want.

Reply:
I used null assertion in lots of other places btw, mostly because you rely on lots of env variables, I.
I wanted to enable the lint rule that bans null assertions but I'd have to add a LOT of if statements 😛

Reply:
Yes, I saw that you add these assertions - thank you for this - it will make it more error proof. Please add this assertion or make it private method if it's possible. Currently it's public method - someone can call this method and get weird error :)

Reply:
Yes, I saw that you add these assertions - thank you for this - it will make it more error proof. Please add this assertion or make it private method if it's possible. Currently it's public method - someone can call this method and get weird error :)